BASE/Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/TelemetryConfigurationFactoryTest.cs

```diff
@@ -1939,6 +1940,8 @@ private class StubTelemetryChannel2 : ITelemetryChannel, ITelemetryModule
 
             public void Flush() { }
 
+            public Task<bool> FlushAsync() { return Task.Run(() => true); }
```

Comment:
Please change this to `return Task.FromResult(true);`

Reply:
Fixed